README.md

```diff
@@ -2,10 +2,9 @@
 
 ## Description ##
 
-This distribution provides an API for the GeoIP2
-[Precision web services](https://dev.maxmind.com/geoip/geoip2/web-services) and
-[databases](https://dev.maxmind.com/geoip/geoip2/downloadable). The API also
-works with the free [GeoLite2 databases](https://dev.maxmind.com/geoip/geoip2/geolite2/).
+This distribution provides an API for the GeoIP2 and GeoLite2 [Precision web
```

Comment:
Precision here is probably not correct if we're moving GeoLite2 into the sentence.